### DISCUSSION TITLE
Welcome to Wumpus Discussions!

### DISCUSSION THREAD

**Discussion by ilhs1041495** [thewumper/Wumpus#71] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ilhs1096069** [comment]
we are wumping

**Comment by MaximumMaxxx** [comment]
Hi! My name is Max. I lose to wump so hard yippeee
